### PR TITLE
updates to latest kork-cassandra

### PR DESCRIPTION
**Pull request by cfieber**
keyspaces are now initialized in PostConstruct instead of on context refresh

Depends on spinnaker/kork#27

@ajordens ptal

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,6 +54,9 @@ allprojects { project ->
     resolutionStrategy {
       force 'com.google.guava:guava:15.0'
       eachDependency {
+        if (it.requested.group == 'com.netflix.spinnaker.kork') {
```
Comment (cfieber) [inline build.gradle]: this needs to go before finally shipping it... yay library release train...

File: build.gradle [hunk 2]
```diff
@@ -29,13 +29,17 @@ buildscript {
 }
 
 allprojects { project ->
+  repositories {
+    mavenLocal()
```
Comment (robfletcher) [inline build.gradle]: You sure you meant to commit that?

File: build.gradle [hunk 3]
```diff
@@ -29,13 +29,17 @@ buildscript {
...
+  }
+
   apply plugin: 'spinnaker.project'
   apply plugin: 'groovy'
 
   group = "com.netflix.spinnaker.front50"
 
   spinnaker {
-    dependenciesVersion = "0.19.0"
+    dependenciesVersion = "0.23.0"
```
Comment (ajordens) [inline build.gradle]: Care to bump this up to 0.25.0.

File: build.gradle (hunk 1 above)
Comment (ajordens) [inline build.gradle]: This can go once you've gone to 0.25.0

**Pull request merged**